src/engine/loaders/obj_loader.cpp

```diff
@@ -71,31 +71,140 @@ void ObjLoader::load(MeshFrame *o_result, char *t_filename, float t_scale, u8 t_
             }
             else if (strcmp(lineHeader, "f") == 0)
             {
-                int matches = fscanf(file, "%d/%d/%d %d/%d/%d %d/%d/%d\n",
-                                     &vertexIndex[0], &coordIndex[0], &normalIndex[0],
-                                     &vertexIndex[1], &coordIndex[1], &normalIndex[1],
-                                     &vertexIndex[2], &coordIndex[2], &normalIndex[2]);
-                if (matches != 9)
-                    PRINT_ERR(".obj can't be read by this simple parser. Try exporting with other options");
+
+                int* x;
+                fpos_t start;
+                fgetpos(file,&start);
+                        int matches = fscanf(file, "%d/%d/%d %d/%d/%d %d/%d/%d\n",x,x,x,x,x,x,x,x,x);
+                fsetpos(file,&start);
+                int newerMatches =0;
+
+                switch(matches)
+                {
+                    /* Vs, VTs and VNs all set */
```

Comment:
Please use doxygen style of comments `/**`
https://www.doxygen.nl/manual/docblocks.html

Reply:
Original obj_loader code comments do not follow this standard though?

Reply:
> Original obj_loader code comments do not follow this standard though?

They are
https://github.com/h4570/tyra/blob/ea6d5215d1b0da693ccd0c377c045fad7421a653/src/engine/loaders/obj_loader.cpp#L105

But of course, chances are I haven't noticed it somewhere.